pkg/logs/tailers/file/tailer.go

```diff
@@ -359,13 +363,16 @@ func (t *Tailer) forwardMessages() {
 		if len(output.GetContent()) == 0 {
 			continue
 		}
+
+		// XXX(remy): is it ok recreating a message like this here?
```

Comment:
I think you leaving the following line as is probably answers the question remy had here quite positively. Would it make sense to remove this comment?

Reply:
Ill defer to @remeh for this (I think it's fine to remove)

Reply:
Please, feel free to remove it.
At that time I was curious that we were doing a copy instead of passing the data, but for sure this will work.